.github/dependabot.yml

```diff
@@ -4,6 +4,16 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+    ignore:
+      # For AWS SDK for Rust, ignore all updates
+      - dependency-name: "aws-config"
+      - dependency-name: "aws-endpoint"
+      - dependency-name: "aws-http"
+      - dependency-name: "aws-hyper"
+      - dependency-name: "aws-sig*"
+      - dependency-name: "aws-sdk*"
+      - dependency-name: "aws-smithy*"
+      - dependency-name: "aws-types"
```

Comment:
We could just ignore `aws*` for simplicity, but this should cover our bases (and then some).